cli/.js/src/main/scala/aqua/json/TypeValidator.scala

```diff
@@ -18,19 +18,20 @@ import scala.scalajs.js.JSON
 
 object TypeValidator {
 
-  // Compare and validate type generated from JSON and type from Aqua file.
-  // Also, validation will be success if array or optional field will be missed in JSON type
-  def validateTypes(name: String, lt: Type, rtOp: Option[Type]): ValidatedNec[String, Unit] = {
-    rtOp match {
+  // Compare and validate type from Aqua file and type generated from JSON.
+  // Also, validation will be success if array or optional field will be missed in JSON type.
+  // The result will be invalid if there will be the same type on the left and right.
+  def validateTypes(name: String, aquaType: Type, jsonType: Option[Type], typeDescription: Option[(Type, Type)] = None): ValidatedNec[String, Unit] = {
```

Comment:
What is `typeDescription`? Could you make the common scaladoc for all arguments please